### ISSUE TITLE
Wont Detect Cards

### ISSUE THREAD

**Issue by TomNOTW** [ForOhForError/CustomCardCam#3]
Hey mate, I followed the ReadMe but I couldnt get cards to recognize once they where in a custom set. Custom sets would show up but not do anything

**Comment by ForOhForError** [comment]
Yeah, this was caused by some outdated image uri code in the scryfall library. I've got a fix in the works.
